loadtest/control/simplecontroller/actions.go

```diff
@@ -181,68 +168,83 @@ func (c *SimpleController) reload(full bool) control.UserActionResponse {
 	if len(roleNames) > 0 {
 		_, err = c.user.GetRolesByNames(roleNames)
 		if err != nil {
-			return control.UserActionResponse{Err: err}
+			return control.UserActionResponse{Err: control.NewUserError(err)}
 		}
 	}
 
 	err = c.user.GetWebappPlugins()
 	if err != nil {
-		return control.UserActionResponse{Err: err}
+		return control.UserActionResponse{Err: control.NewUserError(err)}
 	}
 
 	_, err = c.user.GetAllTeams(0, 50)
 	if err != nil {
-		return control.UserActionResponse{Err: err}
+		return control.UserActionResponse{Err: control.NewUserError(err)}
 	}
 
-	if teamId != "" {
-		err = c.user.GetChannelsForTeam(teamId)
+	_, err = c.user.GetTeamsForUser(userId)
+	if err != nil {
+		return control.UserActionResponse{Err: control.NewUserError(err)}
+	}
+
+	team, err := c.user.Store().RandomTeamJoined()
```

Comment:
Yeah .. I was wondering about this too. Preferences never stored the team id. So how does the webapp work ?